### PR TITLE
add voucher col and adjust styles

### LINKED ISSUE #153: Add link to clinic names
We could link the website URL to the clinic name and remove the website column.

### LINKED ISSUE #166: Add table column. Is Voucher Required.
# Create IsVoucherRequired field
It would save time if we knew which clinics need a voucher. 

Let's add a boolean field!

### Acceptance criteria:
- [x] Create field
- [x] Add to cancellation table

### PR DESCRIPTION
**Pull request by theyokohamalife**
## Resolves #166 Resolves #153

Adds true or false value to column if a voucher or ward residency is required.

### Screenshots

Screen Shot 2021-06-26 at 11 50 23 PM (2)

Screen Shot 2021-06-27 at 12 57 16 AM (2)

### CODE REVIEW

File: pages/add-clinic.vue [hunk 1]
```diff
@@ -5,65 +5,87 @@
       v-model="valid"
       lazy-validation
       @submit.prevent="submitData"
+      class="mb-8 mt-8"
```
Comment (Ermish) [inline pages/add-clinic.vue:8]: Can be done in separate PR:

We should make a class Anne relevant to the actual element the. Apply these classes in a sass rule.

File: pages/add-clinic.vue [hunk 2]
```diff
@@ -5,65 +5,87 @@
...
     >
       <div align="center" id="instructions">
         <h2>{{ $t("add-clinic.fillForm") }}</h2>
       </div>
       <v-container>
-        <div align="center" class="red--text text--darken-4">
+        <v-row class="red--text text--darken-4 mb-4 justify-center">
```
Comment (Ermish) [inline pages/add-clinic.vue:14]: Same with these.

File: pages/add-clinic.vue [hunk 3]
```diff
@@ -5,65 +5,87 @@
...
+            filled
+            :rules="websiteRules"
+          ></v-text-field>
+        </v-row>
+        <v-row>
+          <v-col>
+            <v-switch
+              v-model="voucher.required"
+              inset
+              :label="`Voucher required: ${voucher.text}`"
```
Comment (Ermish) [inline pages/add-clinic.vue:67]: Should the 'voucher required' also be in the translation?

File: pages/add-clinic.vue [hunk 4]
```diff
@@ -5,65 +5,87 @@
...
+            ></v-switch>
           </v-col>
         </v-row>
+        <v-row>
+          <v-textarea
+            v-model="note"
+            outlined
+            name="input-7-4"
+            :label="$t('add-clinic.additionalInfo')"
+            placeholder="Share helpful or important details"
```
Comment (Ermish) [inline pages/add-clinic.vue:86]: Should move placeholder to translation

File: pages/add-clinic.vue [hunk 5]
```diff
@@ -5,65 +5,87 @@
...
-              v-model="note"
-              outlined
-              name="input-7-4"
-              :label="$t('add-clinic.additionalInfo')"
-              placeholder="Share helpful or important details"
-            ></v-textarea>
+            <v-switch
+              v-model="wardResidency.required"
+              inset
+              :label="`Ward residency required: ${wardResidency.text}`"
```
Comment (Ermish) [inline pages/add-clinic.vue:75]: Same with all the text here for translations

File: pages/add-clinic.vue [hunk 6]
```diff
@@ -116,6 +146,20 @@ export default {
...
+    handleVoucherSwitch() {
+      if (this.voucher.required) {
+        this.voucher.text = "Yes";
+      } else {
+        this.voucher.text = "No";
+      }
+    },
+    handleWardResidencySwitch() {
+      if (this.wardResidency.required) {
+        this.wardResidency.text = "Yes";
```
Comment (Ermish) [inline pages/add-clinic.vue:158]: Should put the yes / no in Translations

**Review by Ermish — APPROVED** [review]
Could make these changes in this PR or a separate if we make a new issue.

File: pages/add-clinic.vue (hunk 3 above)
Comment (theyokohamalife) [reply pages/add-clinic.vue:67]: Yeah so all users know what the switch is for.

File: pages/add-clinic.vue (hunk 6 above)
Comment (theyokohamalife) [reply pages/add-clinic.vue:158]: Oops! I forgot!

**Pull request merged**